### DISCUSSION TITLE
POINT data type doesn't exist?

### DISCUSSION THREAD

**Discussion by sergeda** [cockroachdb/cockroach#122737] [Q&A] [answered]
According to the documentation https://www.cockroachlabs.com/docs/stable/point
CRDB supports `POINT` data type together with other types like `POINTM` etc.
But when I specify `"location" POINT NOT NULL` in my table definition I'm getting 
> syntax error: unimplemented

Why this doesn't work?

**Comment by yuzefovich** [comment, marked as answer]
IIUC `POINT` type is a concrete implementation of `GEOMETRY` type, so you need to specify something like `"location" GEOMETRY NOT NULL` and then you can insert values like `POINT(0 0)'::GEOMETRY` into that column.
